### ISSUE TITLE
Test names

### ISSUE THREAD

**Issue by davidbilik** [davidvavra/showkase-screenshot-tests#11]
Hi, I love the approach of integrating Showkase with any screenshoting lib. We've added this to our project and one thing I don't particularly like are the names of the generated screenshot files. They contain an index of a screenshot in the list from Showkase and it basically  re-generates all screenshots when new `@Preview` method is introduced and the order changes. I've tried to check the TestParameterInjector if there is a way to modify this or the paparazzi file name generation but with no luck. What was your approach to this? 

Thanks!

**Comment by davidvavra** [comment]
I don't see an issue with a screenshot index in the name. This is an example of a file name generated by code in this repo:

`me.vavra.screenshots_ComposePaparazziTests_preview_tests[Greetings:Feature module greeting,1.5,dark].png`

It's maybe too long and contains unneccerary stuff, but there is no screenshot index. Btw the file name is partially driven by the `toString()` method in [this file](https://github.com/davidvavra/showkase-screenshot-tests/blob/main/screenshots/src/test/java/me/vavra/screenshots/ComposePaparazziTests.kt).

**Comment by davidbilik** [comment]
Hmm, it's strange, because my screenshot name is called
`com.xxx.screenshots_ComposePreviewsTests_snapshotPreview[1.Room - AVB:BroadcastMediaControlsPreview,1.1.0,1.light].png` and `com.xxx.screenshots_ComposePreviewsTests_snapshotPreview[2.Room - AVB:LiveBroadcastBottomSheetHeaderPreview,1.1.0,1.light].png`

and the index before the group name makes the mess. If the index would not be there then I'm all set

**Comment by davidvavra** [comment]
Try to look for differences between your implementation and this repo

**Comment by davidbilik** [comment]
I've finally figured it out. Once the values provided by `TestParameterValuesProvider` does not have distinct `toString` representation the framework puts the index before the parameter value. Our project had a couple of previews with a `@PreviewParameter` and then is the `componentName` of the `ShowkaseBrowserComponent` same for all variants of previews. I've added some logic around creating a list of components and their toString implementation and now no indices 😌
